### PR TITLE
FeedbackSessions: refine tooltip for 'Publish manually' option #1085

### LINKED ISSUE #1085: FeedbackSessions: refine tooltip for 'Publish manually' option
_From [dam...@gmail.com](https://code.google.com/u/114919422028123491955/) on August 03, 2013 17:33:12_

The current tooltip is a bit difficult to understand.

_Original issue: http://code.google.com/p/teammatespes/issues/detail?id=1086_

### PR DESCRIPTION
**Pull request by XianHuiKoh**
Fixes #1085

### CODE REVIEW

File: src/test/resources/pages/instructorFeedbackAddSuccess.html [hunk 1]
```diff
@@ -344,7 +344,7 @@ <h5 class="col-sm-4">
                                         </div>
                                     </div>
                                     <div class="row radio">
-                                        <div data-original-title="Select this option if you intend to manually publish the session later on." class="col-md-4" title="" data-toggle="tooltip" data-placement="top">
+                                        <div data-original-title="Select this option if you intend to manually publish the feedback responses for this session later on." class="col-md-4" title="" data-toggle="tooltip" data-placement="top">
```
Comment (damithc) [inline src/test/resources/pages/instructorFeedbackAddSuccess.html]: use 'responses' instead of 'feedback responses'. In the user's eye, not all sessions are 'feedback sessions' although in the code they are all considered `FeedbackSession`s. For the user, they can be surveys, quizzes, etc.

File: .launches/Staging tests.launch [hunk 2]
```diff
@@ -5,7 +5,7 @@
 </listAttribute>
 <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="org.testng.remote.RemoteTestNG"/>
 <stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="-testnames sequential-ui-tests,parallel-ui-tests,occassional-tests"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="teammates"/>
+<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="Teammates"/>
```
Comment (damithc) [inline .launches/Staging tests.launch]: Remove unrelated changes.

**Comment by LowWeiLin** [comment]
There are still some files/changes that should not be included in the pull request. You can see the changes using the files changed tab.
image

File: src/test/resources/pages/instructorFeedbackDoneEditing.html [hunk 3]
```diff
@@ -358,7 +358,11 @@ <h5 class="col-sm-2">
                                     </div>
                                 </div>
                                 <div class="row radio">
+<<<<<<< HEAD
+                                    <div data-placement="top" data-toggle="tooltip" title="" class="col-md-4" data-original-title="Select this option if you intend to manually publish the responses for this session later on.">
+=======
```
Comment (kanghj) [inline src/test/resources/pages/instructorFeedbackDoneEditing.html]: You need to fix your merge conflicts first. Check that the tests pass.

**Comment by kanghj** [comment]
Looks good. Good work.
